RELEASE_NOTES.md

```diff
@@ -2,7 +2,7 @@
 
 ## Summary
 
-<!-- Here goes a general summary of what this release is about -->
+Updated frequenz-api-common dependency from v0.5.1 to v0.6.0
 
 ## Upgrading
```

Comment:
Since this is client-facing, I would also suggest add the update entry here, along with a link to the release notes of common-API v0.6.0: https://github.com/frequenz-floss/frequenz-api-common/releases/tag/v0.6.0

Reply:
Good idea, thanks. I added as suggested.